### DISCUSSION TITLE
Focus remains in previous window

### DISCUSSION THREAD

**Discussion by smpallen99** [zed-industries/zed#8551] [Feature Requests]
I'm posting this here since I can't reproduce the issue, but thought I'd mention it.

I had 3 to 5 different projects open in separate zed instances (windows??). The 2 in question here were on different monitors (for what that's worth).

I clicked into the 2nd project window and started typing, but the text was inserted into the 1st instance. I swapped back to the 1st instance, escaped out of insert mode, and clicked into the 2nd instance. Clicked a couple times to make sure and started typing. Same behaviour. 

I was in pressed for time and did not investigate futher. I closed the 2nd instance and opened the file in neovide to finish my work. 

I'll keep an eye out for the issue again gather more information.

Zed: v0.124.6 (Zed Preview)
OS: macOS 13.3.1
Memory: 32 GiB
Architecture: x86_64

**Comment by mrnugget** [comment]
Can you create a ticket for this? I ran into the same issue, so it would be nice to have a single ticket in which we can collect information about it.

**Reply by smpallen99** [reply]
Going to do that now. I have it in that state right now. 2nd time for me now.
